packages/ui/src/theme/css/component/avatar.scss

```diff
@@ -126,4 +141,13 @@
     object-fit: cover;
     display: block;
   }
+
+  &__loader {
+    position: absolute;
```

Comment:
`position: absolute` feels wrong here, what does it provide?

Reply:
If you look at the gif in the description, this allows the circular loader to be placed around the avatar component

Reply:
Got it, makes sense now